### PR TITLE
NBug issue template: Add default text

### PR DESCRIPTION
**Pull request by gerhardol**
As suggested in https://github.com/gitextensions/gitextensions/issues/6607#issuecomment-494683221
and https://github.com/gitextensions/NBug/pull/18

The intention is to have something the bot can close automatically without having the submitter or other users "offended".

### CODE REVIEW

File: NBug/Core/Util/ErrorReportMarkDownBodyBuilder.cs [hunk 1]
```diff
@@ -29,6 +29,8 @@ public string Build(Exception exception, string additionalInfo)
     :warning: Review existing issues to see whether someone else has already reported your issue.
 -->
 
+The sections below must be filled in and this text must be removed or the issue will be closed.
```
Comment (RussKie) [inline NBug/Core/Util/ErrorReportMarkDownBodyBuilder.cs]: do we want to add :warning: icon?

**Review by RussKie — APPROVED** [review]

File: NBug/Core/Util/ErrorReportMarkDownBodyBuilder.cs (hunk 1 above)
Comment (gerhardol) [reply NBug/Core/Util/ErrorReportMarkDownBodyBuilder.cs]: Good idea. 
More important: is this text enough for the bot to close or at least add a label?

Comment (RussKie) [reply NBug/Core/Util/ErrorReportMarkDownBodyBuilder.cs]: The dup bot is looking at the title.
We can build another bot that would close issues as incorrectly filled. This text can be used as a marker.

**Comment by RussKie** [comment]
The dup bot is looking at the title.
We can build another bot that would close issues as incorrectly filled.
This text can be used as a marker.

**Comment by RussKie** [comment]
I made a bot to handle that. Have a look at https://github.com/RussKie/Playpen/issues/26

**Pull request merged**